CRM/Export/Form/Select.php

```diff
@@ -488,7 +488,7 @@ public function buildMapping() {
     $mappingTypeId = CRM_Core_OptionGroup::getValue('mapping_type', $exportType, 'name');
```

Comment:
no, but I was limiting to the one function as that requires an extra step / testing